doc/release-notes/10287-use-support-address-in-system-email-text.md

```diff
@@ -0,0 +1,4 @@
+### Notification Email Improvement
+
+The system email text has been improved to use the support email address (`dataverse.mail.support-email`) in the text where it states; 'contact us for support at', instead of the default system email address.
```

Comment:
```suggestion
The system email text has been improved to use the support email address (`dataverse.mail.support-email`) in the text where it states; "contact us for support at", instead of the default system email address.
```